### PR TITLE
AppVeyor: Improvements

### PR DESCRIPTION
**Pull request by Margen67**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -1,20 +1,39 @@
...
+skip_commits:
+  files:
+    - '*travis*'
+    - '*.md'
+    - '.github/**'
+    - '.clang-format'
+    - 'CHANGES'
+    - 'LICENSE'
+
+skip_branch_with_pr: true
```
Comment (endrift) [inline .appveyor.yml:10]: What does this do and what is the justification for it?

File: .appveyor.yml [hunk 2]
```diff
@@ -1,20 +1,39 @@
...
+
+image: Visual Studio 2019
```
Comment (endrift) [inline .appveyor.yml:12]: Why did you move around all of this stuff?

File: .appveyor.yml [hunk 3]
```diff
@@ -1,20 +1,39 @@
...
+
+environment:
+  QTDIR: C:\Qt\5.15\msvc2019_64
```
Comment (endrift) [inline .appveyor.yml:15]: How does pulling this into an env var help?

File: .appveyor.yml [hunk 4]
```diff
@@ -1,20 +1,39 @@
...
+
+cache: C:\Tools\vcpkg
```
Comment (endrift) [inline .appveyor.yml:17]: Why remove the newline? What if I decide to add things to this later?

File: .appveyor.yml [hunk 5]
```diff
@@ -1,20 +1,39 @@
...
+
+platform: x64
+
+configuration: Release
+
 build:
   parallel: true
   project: mGBA.sln
+
+test: off
```
Comment (endrift) [inline .appveyor.yml:39]: Why turn off testing?

**Review by endrift — COMMENTED** [review]
Very little of this is obviously justifiable on its own, and you have not presented any justification beyond "improvements"

File: .appveyor.yml (hunk 4 above)
Comment (Margen67) [reply .appveyor.yml:17]: Reduces line count. Using multi-line syntax for one line is unnecessary.

File: .appveyor.yml (hunk 1 above)
Comment (Margen67) [reply .appveyor.yml:10]: This option prevents AppVeyor from triggering two builds for a branch with an open PR.

File: .appveyor.yml (hunk 4 above)
Comment (endrift) [reply .appveyor.yml:17]: Line count is not exactly a goal here. The "what if I decide to add things to this later" trumps "well maybe we can save a line"

**Comment by endrift** [comment]
I can reopen this if you explain better the reasoning. I closed it since you weren't replying.
